diff --git a/src/usbd/usbd_conf.c b/src/usbd/usbd_conf.c
index 172a5add..5eccf4d6 100644
--- a/src/usbd/usbd_conf.c
+++ b/src/usbd/usbd_conf.c
@@ -421,7 +421,7 @@ USBD_StatusTypeDef USBD_LL_Init(USBD_HandleTypeDef *pdev)
         hpcd_USB_OTG_FS.Init.low_power_enable        = DISABLE;
         hpcd_USB_OTG_FS.Init.lpm_enable              = DISABLE;
         hpcd_USB_OTG_FS.Init.battery_charging_enable = ENABLE;
-        hpcd_USB_OTG_FS.Init.vbus_sensing_enable     = ENABLE;
+        hpcd_USB_OTG_FS.Init.vbus_sensing_enable     = DISABLE;
         hpcd_USB_OTG_FS.Init.use_dedicated_ep1       = DISABLE;
         if(HAL_PCD_Init(&hpcd_USB_OTG_FS) != HAL_OK)
         {
